cartridges/rvw_dev_console/cartridge/client/default/js/vue/components/App.vue

```diff
@@ -777,6 +735,26 @@ export default {
                 this.promptFile();
             }
         },
+        showMessage (type, message, callback) {
```

Comment:
@z1haze added this to reduce the duplicate code that was showing toast messages, now we can just call a single function and not repeat the same code over and over ( which I was doing ).